.github/workflows/build.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           export PATH=$GOROOT/bin:$PATH
           # rust dependencies
           export CARGO_HOME="$HOME/.cargo"
-          curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain 1.68.2
+          curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain 1.75.0
```

Comment:
This change was added because CI uses stable and matt was getting an unstable error from one of the newer libraries for a feature which should be stable as of 1.75. (1.68 is quite old)